packages/dom/src/dom/is-html-input-element.js

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable jsdoc/valid-types */
+/**
+ * @param {Element | null | undefined} element
```

Comment:
I [mentioned above](https://github.com/WordPress/gutenberg/pull/30386/files#r607019420), but it seems like this could accept values of `Node` type:
```suggestion
 * @param {Node | null | undefined} element
```